### ISSUE TITLE
Update openmm-ml interface with NequIP to work with newer pytorch and CUDA versions

### ISSUE THREAD

**Issue by asoyemi1** [openmm/openmm-ml#95]
Hi developers,

I have been using NequIP in OpenMM with the following versions of pytorch and cuda

pytorch=1.11
pytorch-cuda=11.8
cudatoolkit=11.8

I recently tried to OpenMM/OpenMM-ML with version 2.5 of pytorch and cuda12 and received the following warnings while the MD does not actually run

`[W416 16:00:18.340154879 LinearAlgebra.cpp:3066] Warning: at::frobenius_norm is deprecated and it is just left for JIT compatibility. It will be removed in a future PyTorch release. Please use `linalg.vector_norm(A, 2., dim, keepdim)` instead (function operator())
[W416 16:00:22.069392359 autograd_not_implemented_fallback.cpp:62] Warning: neighbors::getNeighborPairs: an autograd kernel was not registered to the Autograd key(s) but we are trying to backprop through it. This may lead to silently incorrect behavior. This behavior is deprecated and will be removed in a future version of PyTorch. If your operator is differentiable, please ensure you have registered an autograd kernel to the correct Autograd key (e.g. DispatchKey::Autograd, DispatchKey::CompositeImplicitAutograd). If your operator is not differentiable, or to squash this warning and use the previous behavior, please register torch::CppFunction::makeFallthrough() to DispatchKey::Autograd. (function warnAutogradNotImplemented)
`
Are there any plans to update OpenMM-NequIP to work with newer pytorch and cuda?

Thank you!

**Comment by peastman** [comment]
OpenMM works fine with newer CUDA and PyTorch.  That warning is talking about the internals of the specific model you're using.  You need to talk to the NequIP developers about updating it.

That was just a warning, not an error.  It shouldn't prevent the simulation from running.  Was there also an error?
